source/docs/software/driverstation/driver-station.rst

```diff
@@ -121,3 +121,60 @@ Example: The image above shows 4 devices:
 - A Locked, but disconnected "MadCatz GamePad (Controller)" which is a MadCatz Xbox 360 Controller
 
 In this example, unplugging the Logitech Extreme 3D joystick will result in the F310 Gamepad moving up to slot 1. Plugging in the MadCatz Gamepad (even if the devices in Slots 1 and 2 are removed and those slots are empty) will result in it occupying Slot 3.
+
+CAN\\Power Tab
+--------------
+
+.. image:: images/driver-station/ds-can-power-tab.png
+
+
+
+The last tab on the left side of the DS is the CAN\Robot Power Tab. This tab contains information about the power status of the roboRIO and the status of the CAN bus:
+
+1. Comms Faults - This indicates the number of Comms faults that have occurred since the DS has been connected
+2. 12V Faults - This indicates the number of input power faults (Brownouts) that have occurred since the DS has been connected
+3. 6V/5V/3.3V Faults - This indicates the number of faults (typically cause by short circuits) that have occurred on the User Voltage Rails since the DS has been connected
```

Comment:
```suggestion
3. 6V/5V/3.3V Faults - Indicates the number of faults (typically cause by short circuits) that have occurred on the User Voltage Rails since the DS has been connected
```